diff --git a/node_modules/expo-ads-admob/android/build.gradle b/node_modules/expo-ads-admob/android/build.gradle
index 1a24e41..45c7aee 100644
--- a/node_modules/expo-ads-admob/android/build.gradle
+++ b/node_modules/expo-ads-admob/android/build.gradle
@@ -22,7 +22,7 @@ buildscript {
 
 // Creating sources with comments
 task androidSourcesJar(type: Jar) {
-  classifier = 'sources'
+  archiveClassifier = 'sources'
   from android.sourceSets.main.java.srcDirs
 }
 
