diff --git a/android/src/main/java/com/swmansion/reanimated/keyboardObserver/ReanimatedKeyboardEventListener.java b/android/src/main/java/com/swmansion/reanimated/keyboardObserver/ReanimatedKeyboardEventListener.java
index d359217..c48616f 100644
--- a/android/src/main/java/com/swmansion/reanimated/keyboardObserver/ReanimatedKeyboardEventListener.java
+++ b/android/src/main/java/com/swmansion/reanimated/keyboardObserver/ReanimatedKeyboardEventListener.java
@@ -64,7 +64,7 @@ public class ReanimatedKeyboardEventListener {
           }
           int paddingTop = insets.getInsets(WindowInsetsCompat.Type.systemBars()).top;
           View content =
-              rootView.getRootView().findViewById(com.swmansion.reanimated.R.id.action_bar_root);
+              rootView.getRootView();
 
           FrameLayout.LayoutParams params =
               new FrameLayout.LayoutParams(
@@ -149,7 +149,7 @@ public class ReanimatedKeyboardEventListener {
     ViewCompat.setOnApplyWindowInsetsListener(getRootView(), null);
     ViewCompat.setWindowInsetsAnimationCallback(getRootView(), null);
     View content =
-        getRootView().getRootView().findViewById(com.swmansion.reanimated.R.id.action_bar_root);
+        rootView.getRootView();
 
     FrameLayout.LayoutParams params =
         new FrameLayout.LayoutParams(
